specifyweb/frontend/js_src/lib/components/QueryBuilder/Results.tsx

```diff
@@ -270,19 +271,18 @@ export function QueryResults(props: QueryResultsProps): JSX.Element {
         // REFACTOR: turn this into a reusable table component
         className={`
           grid-table auto-rows-min
-          grid-cols-[repeat(var(--meta-columns),min-content)_repeat(var(--columns),auto)]
           overflow-auto rounded
           ${tableClassName}
           ${showResults ? 'border-b border-gray-500' : ''}
-       `}
+        `}
         ref={scrollerRef}
         role="table"
-        style={
-          {
-            '--columns': visibleFieldSpecs.length,
-            '--meta-columns': (showLineNumber ? 1 : 0) + (hasIdField ? 2 : 0),
-          } as React.CSSProperties
-        }
+        style={{
+          gridTemplateColumns: [
+            ...new Array(metaColumns).fill('min-content'),
```

Comment:
don't use new Array please (we should have an eslint-rule against that)
it's bug prone - and Array.from was added to javascript to replace it - `Array.from({length: metaColumns})`
it's bug prone because `Array.from(2)` creates array with two items, but `Array.from(2, 1)` creates an `[2, 1]` array